.github/FUNDING.yml

```diff
@@ -1,3 +1,3 @@
 # These are supported funding model platforms
 
-github: [paulcsmith]
+github: [paulcsmith, jwoertink]
```

Comment:
Thanks 😅  ❤️  Maybe we should just add everyone on the core team so it doesn't look like the "paul and jeremy show"? I don't want to give the wrong impression that people can support me, but not you. Not sure if @edwardloveall or @matthewmcgarvey have sponsor links... Maybe it'll do a silent fail but magically work if they add sponsor links?

Reply:
I'm okay without sponsorship. Thank you for thinking of me!